package.json

```diff
@@ -37,29 +37,32 @@
     "README.md"
   ],
   "peerDependencies": {
-    "@pixi/constants": "^5.0.0",
-    "@pixi/core": "^5.0.0",
-    "@pixi/display": "^5.0.0",
-    "@pixi/graphics": "^5.0.0",
-    "@pixi/math": "^5.0.0",
-    "@pixi/sprite": "^5.0.0"
+    "@pixi/constants": "^5.4.0-rc.3",
+    "@pixi/core": "^5.4.0-rc.3",
+    "@pixi/display": "^5.4.0-rc.3",
+    "@pixi/graphics": "^5.4.0-rc.3",
+    "@pixi/math": "^5.4.0-rc.3",
+    "@pixi/sprite": "^5.4.0-rc.3",
+    "@pixi/utils": "^5.4.0-rc.3"
   },
   "devDependencies": {
     "@microsoft/api-extractor": "^7.9.4",
-    "@pixi-build-tools/rollup-configurator": "^1.0.3",
-    "@pixi/canvas-renderer": "^5.3.3",
-    "@pixi/constants": "^5.3.3",
-    "@pixi/core": "^5.3.3",
-    "@pixi/display": "^5.3.3",
-    "@pixi/graphics": "^5.3.3",
-    "@pixi/math": "^5.3.3",
-    "@pixi/sprite": "^5.3.3",
+    "@pixi-build-tools/rollup-configurator": "~1.0.11",
+    "@pixi/canvas-renderer": "5.4.0-rc.3",
+    "@pixi/constants": "5.4.0-rc.3",
+    "@pixi/core": "^5.4.0-rc.3",
+    "@pixi/display": "^5.4.0-rc.3",
+    "@pixi/graphics": "^5.4.0-rc.3",
+    "@pixi/math": "^5.4.0-rc.3",
+    "@pixi/sprite": "^5.4.0-rc.3",
+    "@pixi/utils": "^5.4.0-rc.3",
     "@rollup/plugin-node-resolve": "^8.4.0",
+    "@types/eventemitter3": "^2.0.2",
     "del": "^2.2.0",
     "jsdoc": "^3.4.0",
     "mkdirp": "^0.5.1",
     "parallelshell": "^2.0.0",
-    "pixi.js": "^5.3.3",
+    "pixi.js": "^5.4.0-rc.3",
```

Comment:
Can this be removed if you're using the leaf packages?

Reply:
👍